client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/linux-frameworks/linux-frameworks.component.ts

```diff
@@ -113,44 +107,61 @@ export class LinuxFramworksComponent implements OnDestroy {
         };
       });
 
-      this.wizard.buildSettings
-        .get('applicationFramework')
-        .valueChanges.takeUntil(this._ngUnsubscribe$)
-        .subscribe(val => {
-          this.setupValidators(val);
-        });
+      this.setupValidators();
       this.wizard.siteArmObj$.takeUntil(this._ngUnsubscribe$).subscribe(site => {
         const linuxFxVersionObj = site.properties.siteProperties.properties.find(x => x.name === 'LinuxFxVersion');
         if (linuxFxVersionObj) {
           const linuxFxVersion = linuxFxVersionObj.value.split('|');
           const stack = linuxFxVersion[0];
           const version = linuxFxVersion[1];
-          this.selectedFramework = stack;
+          switch (stack.toLowerCase()) {
+            case 'node':
```

Comment:
what happens when it is neither

Reply:
The we do not set any value for the framework, and the user has to select the framework. Without selecting, the user is unable to go forward

Reply:
Ok. Make sense. So user is able to edit these?

Reply:
It is a drop down and hence user can select from the supported frameworks. 

The code here tries to auto select framework from the available list, based on the current linux webapp framework.